### DISCUSSION TITLE
Support `output-location: column` in HTML documents

### DISCUSSION THREAD

**Discussion by maxdrohde** [quarto-dev/quarto-cli#3151] [Feature Requests]
In the `reveal.js` format, we can use the `output-location: column` YAML option to show the output in the column next to the code, instead of the standard location below the code (see: https://quarto.org/docs/presentations/revealjs/#output-location)

It seems to me like this is only possible in the `reveal.js` format. It would be great to have this supported in HTML documents as well.

**Comment by maxdrohde** [comment]
To clarify, I see that this is supported for plots and tables using `column: margin`. But the behavior is different for text output. I would like all the code to be in the main column, and all the output in the margin.

## Example

````
```{r}
#| column: margin
plot(1:5)
```

```{r}
#| column: margin
print(1:5)
```
````

### Output
<img width="1016" alt="Screen Shot 2022-11-02 at 18 49 36" src="https://user-images.githubusercontent.com/25734232/199623411-2a556696-60e8-4e06-9176-b536135bbba1.png">

**Reply by dragonstyle** [reply]
This definitely makes sense- I created an issue to track making this change!

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/3550
